.github/workflows/adhoc_create_release_notes.yml

```diff
@@ -0,0 +1,21 @@
+name: Adhoc create confluence release notes
+
+on:
+  workflow_dispatch:
```

Comment:
Should this have something else after it?

Reply:
no - it just means it can be triggered from actions page

Reply:
and there are no input params to this workflow